src/math/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-add_local_sources(sof numbers.c trig.c)
+add_local_sources(sof numbers.c trig.c decibels.c)
```

Comment:
Do we need this if we are passing raw values ?

Reply:
No, it will be removed from this patch for FW.

Reply:
Just an update to my above comment - I made that conclusion too quickly and have not proceed to remove the dB library function.